### PR TITLE
SMS-6066 - Carrier details in message object

### PR DESCRIPTION
**Pull request by NirmitiJain**

### CODE REVIEW

**Review by prashantjha-plivo — APPROVED** [review]
lgtm

**Review by prashantjha-plivo — COMMENTED** [review]
build is failing, please check

**Review by prashantjha-plivo — APPROVED** [review]

File: src/Plivo/Resource/Message/Message.cs [hunk 1]
```diff
@@ -105,6 +105,25 @@ public class Message : Resource
...
+        /// </summary>
+        /// <value>The carrier_fees_rate.</value>
+        public string CarrierFeesRate { get; set; }
+
+
+        /// <summary>
+        /// Gets or sets the carrier_fees.
+        /// </summary>
+        /// <value>The carrier_fees.</value>
+        public string CarrierFees { get; set; }
```
Comment (renoldthomas-plivo) [inline src/Plivo/Resource/Message/Message.cs:125]: These fields should be added in: **public override string ToString()**
as well...

"CarrierFees: " + CarrierFees + "\n" +
"CarrierFeesRate: " + CarrierFeesRate + "\n" +
"DestinationNetwork: " + DestinationNetwork + "\n";

**Review by renoldthomas-plivo — CHANGES_REQUESTED** [review]

File: src/Plivo/Resource/Message/Message.cs (hunk 1 above)
Comment (narayana-plivo) [reply src/Plivo/Resource/Message/Message.cs:125]: @renoldthomas-plivo done

**Review by renoldthomas-plivo — APPROVED** [review]
LGTM

**Pull request merged**